lib/shared/bucketing-assembly-script/assembly/types/target.ts

```diff
@@ -131,6 +133,13 @@ export class AudienceFilterOrOperator extends JSON.Value {
             this.values = null
         }
 
+        const _audiencesArr = filter.getArr('_audiences')
```

Comment:
isnt there already a helper for this block? like getArrayFromJSONOptional or something?

Reply:
It doesnt look like that particular helper is a thing. I could make it though

Reply:
looks like it would be useful

Reply:
yea, added